cmd/qat_plugin/README.md

```diff
@@ -1,32 +1,38 @@
-# Build and test Intel QuickAssist (QAT) Device Plugin for Kubernetes
+# Build and test Intel® QuickAssist Technology (QAT) device plugin for Kubernetes
 
-### Requirements/Prerequisites
-* Ensure that DPDK drivers are loaded and ready to be used. Please refer to the following for more information on drivers:
-  * [DPDK Getting Started Guide for Linux](https://doc.dpdk.org/guides/linux_gsg/index.html)
-  * [Linux Drivers](http://dpdk.org/doc/guides/linux_gsg/linux_drivers.html)
-* Ensure that the QuickAssist SR-IOV virtual functions are configured. Verify this by running:
+### Prerequisites
+* Data Plane Development Kit (DPDK) drivers must be loaded and configured. For more information, refer to:
+    * [DPDK Getting Started Guide for Linux](https://doc.dpdk.org/guides/linux_gsg/index.html)
+    * [DPDK Getting Started Guide, Linux Drivers section](http://dpdk.org/doc/guides/linux_gsg/linux_drivers.html)
+* QuickAssist SR-IOV virtual functions must be configured. Verify this by running:
 ```
 lspci | grep QAT
 ```
- For installation and configuring of Intel QuickAssist Technology software for Linux refer to the [Intel QuickAssist Technology Software for Linux - Getting Started Guide] (https://01.org/sites/default/files/downloads/intelr-quickassist-technology/336212qatswgettingstartedguiderev003.pdf) on [ https://01.org/intel-quickassist-technology](https://01.org/intel-quickassist-technology)
+* Intel QuickAssist Technology software for Linux must be installed and
+  configured. For more information, refer to:
+    * [Intel QuickAssist Technology Software for Linux - Getting Started Guide](https://01.org/sites/default/files/downloads/intelr-quickassist-technology/336212qatswgettingstartedguiderev003.pdf)
+    * [Intel QuickAssist Technology on 01.org](https://01.org/intel-quickassist-technology)
 
-### Get source code
+### Get source code:
 ```
 $ mkdir -p $GOPATH/src/github.com/intel/
 $ cd $GOPATH/src/github.com/intel/
 $ git clone https://github.com/intel/intel-device-plugins-for-kubernetes.git
 ```
-### Make sure kubelet socket exists in /var/lib/kubelet/device-plugins/
+
+### Verify kubelet socket exists in /var/lib/kubelet/device-plugins/ directory:
 ```
 $ ls /var/lib/kubelet/device-plugins/kubelet.sock
 /var/lib/kubelet/device-plugins/kubelet.sock
 ```
-### Build QAT device plugin
+
+### Build QAT device plugin:
 ```
 $ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
 $ make qat_plugin
 ```
```

Comment:
@swatisehgal  The other plugin readmes have this order: Get source code, Build plugin, Verify kubelet socket, etc.
Is it ok to switch these 2 steps (Verify kubelet socket and Build QAT device plugin)?

Reply:
Yes, no problem, we can be consistent with other plugins.

Reply:
@swatisehgal   Done, please review the latest file.